### PR TITLE
feat(replays): Link to replays from the Event>Tag Details table

### LINKED ISSUE #41566: `/issues/:project/tags/:tagName=replayId/` Links to events instead of replays
Example: https://dev.getsentry.net:7999/organizations/sentry-emerging-tech/issues/3745109687/tags/replayId/?project=6273278

Image

The link in the image is pointing to the Event Details page, it should go to Replay Details.

### PR DESCRIPTION
**Pull request by ryan953**
This adds a Replay button, pushed out to the right of the value, on the Tags table.

Screen Shot 2022-11-18 at 12 11 21 PM

The existing blue link points to an Event/Transaction Search page, where we find all events that have the same tag value. For replayId it's not super useful generally... most replays won't have many errors, but they will have lots of transactions. So clicking that link gets you something.

The other direction folks might want to do is to see the replay itself, which is what we've added now.

Fixes #41566

### CODE REVIEW

File: static/app/views/organizationGroupDetails/groupTagValues.tsx [hunk 1]
```diff
@@ -280,7 +292,30 @@ class GroupTagValues extends AsyncComponent<
...
+    pathname: `/organizations/${orgId}/replays/${replaySlug}/`,
+    query: {
+      referrer,
+    },
+  };
+  return (
+    <RightAligned>
+      <Tooltip title={t('View Replay')}>
+        <Link to={target}>
+          <Button size="xs">
```
Comment (billyvg) [inline static/app/views/organizationGroupDetails/groupTagValues.tsx:311]: Should this have an aria label?

**Review by billyvg — APPROVED** [review]
It looks a bit awkward right aligned, it's kinda just floating in the middle of the space it looks like. Where else do we have the `<icon>` + `<replay id>` pattern?

**Comment by benvinegar** [comment]
At low browser viewport widths, I'd expect Replay IDs to condense into short IDs ... but I'm not sure if that pattern exists elsewhere or I'm just making it up.

**Comment by ryan953** [comment]
> At low browser viewport widths, I'd expect Replay IDs to condense into short IDs ... but I'm not sure if that pattern exists elsewhere or I'm just making it up.

When i size things down the table eventually starts to side-scroll, it's just enough space for the full id to appear.
Here's a comparison of truncated `url` tag values, and the replayId with the button just before that scrolling point:

We're a little tight on space left of the button.
SCR-20221121-f99
SCR-20221121-f9x

So i added 4px of left padding to space things out, now there's a chance the id will be truncated too. it's about 50% of the time:
SCR-20221121-fei

**Pull request merged**